rules-tests/DeadCode/Rector/Assign/RemoveUnusedVariableAssignRector/Fixture/skip_in_get_defined_vars.php.inc

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Rector\Tests\DeadCode\Rector\Assign\RemoveUnusedVariableAssignRector\Fixture;
+
+class SkipInCompact
```

Comment:
oh, fixture class need to be renamed:

```suggestion
class SkipInGetDefinedVars
```

Reply:
https://github.com/rectorphp/rector-src/pull/7107

Reply:
would be nice if this was checked in ci

Reply:
That was exists, but removed as become maintenance burden due to many "on purpose" usage to be skipped iirc, manual verification is fine :)